### PR TITLE
Move users and groups to pages/397

### LINKED ISSUE #397: Move the UsersAndGroups component into the `pages` directory and make sure the code matches STYLE_GUIDE.md

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by babalwa-mbolekwa — CHANGES_REQUESTED** [review]
Hey Faith. When users is clicked on the nav it generates an error:
UsersAndGroups

**Review by kaleem99 — APPROVED** [review]

**Review by kaleem99 — CHANGES_REQUESTED** [review]
- Hi Faith looks good but storybook is showing errors pls make sure to update the imports in storybook e.g 
- import UsersAndGroups from "../components/pages/UsersAndGroups/Presentation.jsx";

**Review by babalwa-mbolekwa — APPROVED** [review]

File: frontend/src/components/pages/UsersAndGroups/utils.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ const ERROR_STATUS = "error";
 const WARNING_STATUS = "warning";
 const DEFAULT_STATUS = "default";
 
-export const timeDifferenceInDays = (time) => {
+export function timeDifferenceInDays (time) {
   return Math.floor(Math.abs(new Date() - new Date(time)) / (1000 * 3600 * 24))
 };
 
-export const getPrStatus = (oldestOpenPrTime) => {
+export function getPrStatus (oldestOpenPrTime) {
```
Comment (babalwa-mbolekwa) [inline frontend/src/components/pages/UsersAndGroups/utils.js:14]: Hey Faith please apply destructing here also for the functions' parameters

**Review by babalwa-mbolekwa — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/UsersAndGroups/utils.js (hunk 1 above)
Comment (FaithMo) [reply frontend/src/components/pages/UsersAndGroups/utils.js:14]: Hi Babalwa. I don't think I need to destructure this. This is the utils.js file

**Review by babalwa-mbolekwa — APPROVED** [review]

**Review by kaleem99 — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**